data/testsuite

```diff
@@ -498,3 +498,229 @@ dalla bat kææwa
 dalla bat kææwa
 dog.NOM rice.NOM eat.PST
 `The dog ate rice`
+
+### Agreement in Coordinated NPs
+
+# 1 NP Coordination
+Source: author
+Vetted: f
+Judgement: g
+Phenomena: {Coordination Agreement}
+Ranjitui Chitrai bat kan2wa
+Ranjitu-i Chitra-i bat ka-n2-wa
+Ranjit.INDF-CONJ Chitra-CONJ rice eat-NPT-IND
+`Ranjit and Chitra eat rice`
+
+# 2 NP Coordination, incorrect missing junction vowel 'u' following final consonant
+Source: author
+Vetted: f
+Judgement: u
+Phenomena: {Coordination Agreement}
+Ranjiti Chitrai bat kan2wa
+Ranjit-i Chitra-i bat ka-n2-wa
+Ranjit-CONJ Chitra-CONJ rice eat-NPT-IND
+`Ranjit and Chitra eat rice`
+
+# 3 NP Coordination
+Source: author
+Vetted: f
+Judgement: g
```

Comment:
for grammatical examples, we are supposed to put all of the phenomena visible
for ungrammatical, we only put the one that's being tested
so here since it's grammatical, we could put {Coordination Agreement, Tense Aspect Mood}